shared/charts/istio-gke-helper/values.yaml

```diff
@@ -0,0 +1,19 @@
+# List of components to create Pod Disruption Budgets for and their labels
```

Comment:
I don't see where the values of these keys are used (only the key names themselves, as `$key` in the name of each PDB). Was this copied from another module? Perhaps the yaml can be simplified to not be misleading. Or am I missing something?

Reply:
The values are used on lines 16 & 17 of `pod_disruption_budget.yaml ` and are required, as it's not possible to differentiate components based on only one of the labels:
```yaml
      app: {{ .app | quote }}
      istio: {{ .istio | quote }}
```